app/assets/javascripts/nfg_ui/datetimepicker_configuration.js

```diff
@@ -52,11 +53,11 @@ function () {
     value: function defaultFormat() {
 
       if (this.input.data('datetimepicker') == 'datetime') {
-        return 'Y-MM-DD hh:mm A';
+        return 'YYYY-MM-DD hh:mm A';
```

Comment:
Single Y just showed a "Y" on the UI instead of a year.  I changed it to use full year.